tests/network-setups/indy/docker/run_indy_nodes.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/src/indy_projects/ursa/target/release/
+generate_indy_pool_transactions --nodes 4 --clients 5 --nodeNum 1 2 3 4
```

Comment:
With a compose file, we can use environment variables to configure the number of nodes and clients.